index/secondary_index_manager.hh

```diff
@@ -116,6 +132,13 @@ public:
     bool is_index(view_ptr) const;
     bool is_index(const schema& s) const;
     bool is_global_index(const schema& s) const;
+    lw_shared_ptr<stats> get_index_stats(const sstring& index_name) const {
```

Comment:
nitpick: I see the only caller (?) passes _index.metadata().name() to this function. So maybe it would have been more convenient to pass the _index to this function, not the index name?